README.md

```diff
@@ -7,23 +7,22 @@ These are **very simple** custom git commands that enforce the git user to write
 
 ## Installation:
 
-1. Clone this repo, prefferably in your $HOME directory.
-``` git clone https://github.com/fteem/git-semantic-commits ~/.git-semantic-commits ```
+1. Clone this repo, prefferably in your $HOME directory. ```git clone https://github.com/fteem/git-semantic-commits ~/.git-semantic-commits```
 
-2. Add the folder to your $PATH.
-  * if using Bash, open ```~/.bashrc``` with your favourite editor and add this line: <br/> ``` export PATH=$PATH:$HOME/.git-semantic-commits:$PATH ```
-  * if using Zsh, open ```~/.zshrc``` with your favourite editor and add this line: <br/>  ``` export PATH=$PATH:$HOME/.git-semantic-commits:$PATH ```
+2. Install it as a set of bash scripts or git aliases:
+  * bash scripts: ```cd ~/.git-semantic-commits && ./install.sh --scripts```
+  * git aliases ```cd ~/.git-semantic-commits && ./install.sh```
 
-    Tip: If you're using Cygwin, open it and type 'echo $USERPROFILE'. This will show you the location of the $HOME directory.
```

Comment:
No! It's just that with this installer user shouldn't be bothered  about any environment variables anymore, so I decided to edit this. In this version I put it back.

💡 Moreover, now I added relative dir path to all the commands, so when installing automatically it doesn't really matter where user cloned the repo. In the previous version it must've been in $HOME dir.